### ISSUE TITLE
Uncloak sound

### ISSUE THREAD

**Issue by Noname12345GER** [MattJeanes/TARDIS#214]
What if you would be able to play a reversed cloak sound on uncloak?

**Comment by MattJeanes** [comment]
If someone finds me a good enough sound for it sure

**Comment by kennyrkun** [comment]
[I made this. It's probably terrible.](http://www.mehsworld.com/p/kennyrkun/files/phase_disable.wav)

**Comment by doctorwho1200** [comment]
Well, i have one
Took it from the episode and reversed
But i've no idea how to put it in here :P

**Comment by kennyrkun** [comment]
@doctorwho1200 put it in a zip file and upload that, or put it on YouTube and link it.

**Comment by doctorwho1200** [comment]
bah, too complicated. this should work
https://www.dropbox.com/home?preview=uncloak.wav

**Comment by Noname12345GER** [comment]
The Doctor included all sounds in his Extensions for the case of being implemented.

**Comment by MattJeanes** [comment]
@doctorwho1200 yeah thats a cool sound when I get round to putting cloak in the rewrite i'll use that
